GiniVision/Classes/Core/Screens/Document picker/DocumentPickerCoordinator.swift

```diff
@@ -172,6 +177,20 @@ public final class DocumentPickerCoordinator: NSObject {
         
         if #available(iOS 11.0, *) {
             documentPicker.allowsMultipleSelection = giniConfiguration.multipageEnabled
+            
+            // Starting on iOS 11.0, the UIDocumentPickerViewController navigation bar can't almost be customized,
+            // being only possible to customize the tint color. To avoid issues with custom UIAppearance styles,
+            // this is reset to default, saving the current state for further restoring when dismissing.
```

Comment:
Starting **with** iOS 11.0, the UIDocumentPickerViewController navigation bar **almost** can't ~almost~ be customized, ~being~ only **being** possible to customize the tint color. To avoid issues with custom UIAppearance styles, this is reset to default, saving the current state ~for further restoring when dismissing~ in order to restore it during dismissal.